### PR TITLE
docs: IPv6 config examples with content tabs

### PR DESCRIPTION
**Pull request by polarathene**
# Description

For added clarity, a user requested we document the example config snippets instead of only linking external references to them.

Some content has been shuffled around and revised, the [`nginx-proxy` comment link removed](https://github.com/nginx-proxy/nginx-proxy/issues/133#issuecomment-1368745843) (_replaced with simpler `networks` syntax, and same as typical user-defined network creation, reduced to a small tip on `default` overriding_), content tabs for presentation all within a single example admonition.

Resolves: #3422 

## Type of change

- [x] Improvement (non-breaking change that does improve existing functionality)

## Preview

Direct preview link to updated section: https://pullrequest-3436--dms-doc-previews.netlify.app/config/advanced/ipv6/#enable-proper-ipv6-support

### Revised version

Screenshot_20230720_205718

### Prior version

Screenshot_20230720_205825

### CODE REVIEW

**Comment by polarathene** [comment]
@AlperShal hopefully this is what you roughly had in mind :sweat_smile: 

If the content looks a bit too noisy, I could collapse the info and tip admonitions by default to compact the focus on what's important.

File: docs/content/config/advanced/ipv6.md [hunk 1]
```diff
@@ -64,34 +64,96 @@ Enable `ip6tables` support so that Docker will manage IPv6 networking rules as w
...
-Next, configure a network for your container with any of these:
+Next, configure a network with an IPv6 subnet for your container with any of these examples:
 
-- [User-defined networks via `docker network create` or `compose.yaml`][docker-docs-ipv6-create-custom]
-- [Default docker bridge][docker-docs-ipv6-create-default] (_docker CLI only, not helpful for `compose.yaml`_)
-- [Default network for a `compose.yaml`][ipv6-config-example] (_`/etc/docker/daemon.json` settings for default bridge do not apply, instead override the generated `default` network_)
+???+ example "Create an IPv6 ULA subnet"
 
-!!! danger "Do not use `2001:db8:1::/64` for your private subnet"
+    !!! info
```
Comment (polarathene) [inline docs/content/config/advanced/ipv6.md:71]:
```suggestion
    ??? info "About these examples"
```

File: docs/content/config/advanced/ipv6.md [hunk 2]
```diff
@@ -64,34 +64,96 @@ Enable `ip6tables` support so that Docker will manage IPv6 networking rules as w
...
+                networks:
+                  - dms-ipv6
+
+            networks:
+              dms-ipv6:
+                enable_ipv6: true
+                subnet: fd00:cafe:face:feed::/64
+            ```
+
+            !!! tip "Override the implicit `default` network"
```
Comment (polarathene) [inline docs/content/config/advanced/ipv6.md:98]:
```suggestion
            ??? tip "Override the implicit `default` network"
```

File: docs/content/config/advanced/ipv6.md [hunk 3]
```diff
@@ -64,34 +64,96 @@ Enable `ip6tables` support so that Docker will manage IPv6 networking rules as w
...
+
+                You can optionally avoid the service assignment by [overriding the `default` user-defined network that Docker Compose generates](docker-docs-network-compose-default). Just replace `dms-ipv6` with `default`.
+
+                The Docker Compose `default` bridge is not affected by settings for the default `bridge` (aka `docker0`) in `/etc/docker/daemon.json`.
+
+            !!! tip "Using the network outside of this `compose.yaml`"
```
Comment (polarathene) [inline docs/content/config/advanced/ipv6.md:104]:
```suggestion
            ??? tip "Using the network outside of this `compose.yaml`"
```

**Review by polarathene — COMMENTED** [review]
Hiding additional information by default to compact the instructions so they're focused on communicating the important info without excess noise.

Here's an updated screenshot with those changes, along with the other two tab views:

Screenshot_20230721_090014

Screenshot_20230721_090045

Screenshot_20230721_090108

**Comment by polarathene** [comment]
Ready for review, auto-merge is all set to go when another maintainer approves :+1:

**Review by casperklein — APPROVED** [review]

**Pull request merged**